btr-web/btr-main-app/components/individual-person/summary-table/ActionButton.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+type ActionBtnT = {
+  action: (val?: any) => any,
```

Comment:
Should this be 'Function' type don't love using any type if it can be avoided, for actionArg could it seems from the use it should just be number

Reply:
yep, good points I will update